packages/api/src/models/card/card-action.ts

```diff
@@ -7,13 +7,14 @@ export type CardActionType =
   | 'showImage'
   | 'downloadFile'
   | 'signin'
-  | 'call';
+  | 'call'
+  | 'invoke';
 
 export type CardAction = {
   /**
    * The type of action implemented by this button. Possible values include: 'openUrl', 'imBack',
    * 'postBack', 'playAudio', 'playVideo', 'showImage', 'downloadFile', 'signin', 'call',
-   * messageBack', 'openApp'
```

Comment:
eventually we should probably just use the card action types from the cards package here.

Reply:
good idea - will add a ticket for it